.travis.yml

```diff
@@ -1,22 +1,37 @@
 ---
-sudo: false
-
 language: node_js
 node_js:
-  - "4"
   - "6"
-  - "7"
+
+sudo: false
+
+cache:
+  yarn: true
+
+env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
+  - EMBER_TRY_SCENARIO=ember-lts-2.4
```

Comment:
There is no ember specific code here, I don't think we need to test more than one (slowing down CI runs...)